posts/pre-commit-and-ruff/index.md

```diff
@@ -141,6 +141,11 @@ It's also possible to [match filenames on a regex pattern](https://pre-commit.co
 
 We've adopted *pre-commit* and *Ruff* on several projects, including [QGreenland](https://github.com/nsidc/qgreenland), [earthaccess](https://github.com/nsidc/sipn-reanalysis-ingest), [US AON Benefit Tool](https://github.com/nsidc/usaon-vta-survey), and [CFSR Reanalysis Tool](https://github.com/nsidc/sipn-reanalysis-ingest).
 
+This gif and image show an example of these tools being used: 
+![](precommit_ruff.gif)
```

Comment:
It'd be useful to point out ruff is fast, so the output appears very briefly, and if you don't watch closely you might miss it!